website/content/docs/platform/k8s/helm/run.mdx

```diff
@@ -479,8 +492,9 @@ Similar to the standby nodes, the former primary also needs to be unsealed:
 $ kubectl exec -ti <name of pod> -- vault operator unseal
 ```
 
-After a few moments the Vault cluster should elect a new active primary. The Vault
-cluster is now upgraded!
+After a few moments the Vault cluster should elect a new active primary if it did
```

Comment:
I think if the goal is to explain how to use auto upgrades with both strategies, we should have standalone sections for each.  If that's not the goal, can we be more explicit about which strategy these instructions apply to?

Rereading it all I think the goal is to explain how to use auto-upgrade with the OnDelete strategy.  In which case I think maybe we're missing a step or two where we advise them to look at the autopilot state output before deciding to move forward.  For example, once the newly deployed pods have been unsealed, poll sys/storage/raft/autopilot/state until you see that one of the new pods has become a leader and all the old ones are non-voters.  Then you can proceed to delete the old pods.

As I wrote the above and looked closer at the instructions, it's sounding like what's being proposed in the instructions above is actually incompatible with auto-upgrades.  If we're not creating new PVs and adding the new nodes as new cluster members, then I don't think we can use auto-upgrade?  The way it's meant to work is that it waits until there are enough new version nodes have joined and proven themselves stable to replace all the old version nodes, then transfer leadership to one of them, then demote all the old nodes to non-voters, then wait for the user to delete the old nodes from raft config.

Reply:
Okay, I can rework this to add a separate section for `RollingUpdate`, and I'll double check if new PVs are created.

I did test that the this workflow (`OnDelete`, and deleting pods as suggested) worked as expected with Automated Upgrade.